src/main/java/com/salesforce/mirus/metrics/MirrorJmxReporter.java

```diff
@@ -112,6 +149,14 @@ public synchronized void recordMirrorLatency(String topic, long millis) {
     if (sensor != null) {
       sensor.record((double) millis);
     }
+
+    Map<Long, Sensor> bucketSensors = histogramLatencySensors.get(topic);
```

Comment:
While I like how we're explicitly reporting zeros for empty buckets for every topic, I'm concerned it might generate too many metrics and slow down our queries.  Perhaps we should only report non-zero values? I'm not sure if that would be easy, or even possible, but something to consider. Perhaps we can filter out zeros in jmxtrans instead?

Reply:
@pdavidson100 non-zero values you mean we don't report records in the 0m bucket? As this metrics only report every second, I don't think it will generate many metrics.

Reply:
Strange - I made this comment after noticing a lot of zeros in Argus for the PRD test cluster.

Reply:
(unless we need this to be a cumulative histogram)

Reply:
After chatting with Paul I realized that there are advantages in using a cumulative histogram, specifically that adding new buckets in the future won't affect existing queries, so I see now why the current approach is preferable.